src/Sparrow.Server/Utils/ThreadNames.cs

```diff
@@ -0,0 +1,412 @@
+    }
+    
+    public class ThreadDetails
+    {
+        public interface IThreadDetails
+        {
+            public string GetShortName();
+        }
+
+        public class Elector : IThreadDetails
+        {
+            private readonly string _source;
+
+            public Elector(string source)
+            {
+                _source = source;
+            }
+
+            public string GetShortName()
+            {
+                return $"Elector {_source}";
+            }
+        }
+
+        public class EtlProcess : IThreadDetails
+        {
+            private readonly string _tag;
+            private readonly string _name;
+
+            public EtlProcess(string tag, string name)
+            {
+                _tag = tag;
+                _name = name;
+            }
+
+            public string GetShortName()
+            {
+                return $"{_tag} {_name}";
+            }
+        }
+
+        public class Index : IThreadDetails
+        {
+            private readonly string _idxName;
+            private readonly string _dbName;
+
+            public Index(string idxName, string dbName)
+            {
+                _idxName = idxName;
+                _dbName = dbName;
+            }
+
+            public string GetShortName()
+            {
+                return $"IX:{_dbName} {_idxName}";
+            }
+        }
+
+        public class UploadBackupFile : IThreadDetails
+        {
+            private readonly string _dbName;
+            private readonly string _targetName;
+            private readonly string _taskName;
+
+            public UploadBackupFile(string dbName, string targetName, string taskName)
+            {
+                _dbName = dbName;
+                _targetName = targetName;
+                _taskName = taskName;
+            }
+
+            public string GetShortName()
+            {
+                return $"UBF {_dbName} to {_targetName}";
+            }
+        }
+
+        public class DeleteBackupFile : IThreadDetails
+        {
+            private readonly string _dbName;
+            private readonly string _targetName;
+            private readonly string _taskName;
+
+            public DeleteBackupFile(string dbName, string targetName, string taskName)
+            {
+                _dbName = dbName;
+                _targetName = targetName;
+                _taskName = taskName;
+            }
+
+            public string GetShortName()
+            {
+                return $"DBF {_dbName} from {_targetName}";
+            }
+        }
+
+        public class BackupTask : IThreadDetails
+        {
+            private readonly string _dbName;
+            private readonly string _taskName;
+
+            public BackupTask(string dbName, string taskName)
+            {
+                _dbName = dbName;
+                _taskName = taskName;
+            }
+
+            public string GetShortName()
+            {
+                return $"Backup {_dbName}";
```

Comment:
`Bkp` - there is another one with this shortcut?

Reply:
done
BkpTsk